packages/js/components/src/experimental-tree-control/tree-item.scss

```diff
@@ -44,6 +44,7 @@ $control-size: $gap-large;
 		svg.components-checkbox-control__checked,
 		svg.components-checkbox-control__indeterminate,
 		.components-checkbox-control__input[type='checkbox'] {
+			cursor: default;
```

Comment:
First I wasn't so sure about this, but it does make sense given its mostly a UI presentation of a selected item, versus a user having to specifically click on the checkbox.